src/App.jsx

```diff
@@ -20,13 +20,13 @@ function reducer(state, action) {
 function App() {
   const [messages, dispatch] = useReducer(reducer, []);
 
-  const handleNewMessage = (message, toastType) => {
+  const handleNewMessage = useCallback((message, toastType) => {
     dispatch({ type: "add-message", message, toastType });
-  };
+  },[]);
```

Comment:
@Meet-Kalani Add prettier and eslint config in this project to format the code automatically

Reply:
@Meet-Kalani I couldn't find prettier and eslint config files in the project